circe/src/test/scala/org/http4s/circe/CirceSpec.scala

```diff
@@ -83,28 +84,27 @@ class CirceSpec extends JawnDecodeSupportSpec[Json] {
       // From ArgonautSpec, which tests similar things:
       // TODO Urgh.  We need to make testing these smoother.
       // https://github.com/http4s/http4s/issues/157
-      def getBody(body: EntityBody): Array[Byte] = body.runLog.unsafeRun.toArray
-      val req = Request().withBody(Json.fromDoubleOrNull(157)).unsafeRun
-      val body = req.decode { json: Json => Response(Ok).withBody(json.asNumber.flatMap(_.toLong).getOrElse(0L).toString)}.unsafeRun.body
+      def getBody(body: EntityBody[IO]): Array[Byte] = body.runLog.unsafeRunSync.toArray
+      val req = Request[IO]().withBody(Json.fromDoubleOrNull(157)).unsafeRunSync
+      val body = req.decode { json: Json => Response(Ok).withBody(json.asNumber.flatMap(_.toLong).getOrElse(0L).toString)}.unsafeRunSync.body
       new String(getBody(body), StandardCharsets.UTF_8) must_== "157"
     }
   }
 
   "jsonOf" should {
     "decode JSON from a Circe decoder" in {
-      val result = jsonOf[Foo].decode(Request().withBody(Json.obj("bar" -> Json.fromDoubleOrNull(42))).unsafeRun, strict = true)
-      result.value.unsafeRun must_== Right(Foo(42))
+      val result = jsonOf[IO, Foo].decode(Request[IO]().withBody(Json.obj("bar" -> Json.fromDoubleOrNull(42))).unsafeRunSync, strict = true)
```

Comment:
This is an unfortunate consequence of the parameterization: `jsonOf[A]` was a lot prettier than `jsonOf[IO, A]`.  I don't have an easy answer for what to do about it.